iroh-relay/src/server/client.rs

```diff
@@ -18,59 +18,62 @@ use crate::{
         disco,
         relay::{write_frame, Frame, KEEP_ALIVE},
     },
-    server::{
-        actor::{self, Packet},
-        metrics::Metrics,
-        streams::RelayedStream,
-        ClientConnRateLimit,
-    },
+    server::{clients::Clients, metrics::Metrics, streams::RelayedStream, ClientRateLimit},
 };
 
-/// Configuration for a [`ClientConn`].
+/// A request to write a dataframe to a Client
+#[derive(Debug, Clone)]
+struct Packet {
+    /// The sender of the packet
+    src: NodeId,
+    /// The data packet bytes.
+    data: Bytes,
+}
+
+/// Number of times we try to send to a client connection before dropping the data;
+const RETRIES: usize = 3;
```

Comment:
I still find this the strangest feature and am surprised you don't remove it.

Reply:
you are right, I should probably just drop it

Reply:
done